patient-search/ui/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,91 @@
+import { Button, Icon, Table, Pagination } from '@trussworks/react-uswds';
+import React from 'react';
+
+type TableContentProps = {
+    tableHeader?: string;
+    tableSubHeader?: React.ReactNode | React.ReactNode[] | string;
+    tableHead: { name: string; sortable: boolean }[];
+    tableBody: any[];
+    isPagination?: boolean;
+    totalResults?: number;
+    currentPage?: number;
+    handleNext?: (page: number) => void;
+    buttons?: React.ReactNode | React.ReactNode[];
+};
+
+export const TableComponent = ({
+    tableHeader,
+    tableHead,
+    tableBody,
+    isPagination = false,
+    totalResults = 20,
+    currentPage = 1,
+    handleNext,
+    buttons,
+    tableSubHeader
+}: TableContentProps) => {
+    return (
+        <div className="">
```

Comment:
Class name is missing here. can we have a placeholder.

Reply:
> 

Correct we can, will implement